app-backend/ingest/src/main/scala/com/azavea/rf/ingest/model/IngestLayer.scala

```diff
@@ -0,0 +1,19 @@
+package com.azavea.rf.ingest.model
+
+import spray.json._
+import DefaultJsonProtocol._
+
+import java.util.UUID
+
+/** An ingest layer groups together input sources which should be
```

Comment:
I think these docstrings should include parameter definitions and a description of their significance (e.g. not entirely clear what the `id` here is supposed to be an ID of or how it's going to be used.